Dockerfile

```diff
@@ -30,13 +30,8 @@ RUN apt-get update --fix-missing && apt-get install -y \
     libxrender1 \
     wget \
     libevent-dev \
-    build-essential && \
-    rm -rf /var/lib/apt/lists/*
```

Comment:
This was probably here to keep down the size of the image, right? See https://askubuntu.com/questions/179955/var-lib-apt-lists-is-huge for instance. Should we retain this line?

Reply:
Yes we should.  In fact, I didn't mean to remove it.